.travis.yml

```diff
@@ -8,8 +8,8 @@ addons:
     packages:
     - flac
 julia:
-  - 1.0
-  - 1.3
+  - 1.5
+  - nightly
```

Comment:
This is really dumb, but it seems like if I only have a single stage listed here, Travis only performs the `"Documentation"` job and not the actual tests themselves...not sure what the underlying problem is but worked around it via this for now